drivers/Makefile.dep

```diff
@@ -29,6 +29,16 @@ ifneq (,$(filter apa102,$(USEMODULE)))
   FEATURES_REQUIRED += periph_gpio
 endif
 
+ifneq (,$(filter apds99%full,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_gpio_irq
+  USEMODULE += apds99xx
```

Comment:
This is dependency is already set in the `ifneq` block below which should also match the _full case. So:
```suggestion
```

Reply:
Yep, thanks.